Scrubs Tournament Preset update and Weekly Preset update

Jaybone25 opened the pull request:
Scrubs S4 Tournament preset update

Jaybone25 committed:
Scrubs Tourney Preset update

New hint distro and preset for scrubs tournament

Jaybone25 committed:
Fix spacing in default presets

Fixed spacing under allowed tricks

fenhl commented on data/presets_default.json:
```diff
@@ -485,42 +501,29 @@
+      "prelude"
+    ],
+    "ocarina_songs":                           false,
+    "correct_chest_sizes":                     false,
```
These settings seem to have been generated on stable. They need to be updated for settings changes in Dev, such as `ocarina_songs` having multiple options and `correct_chest_sizes` having been replaced with `correct_chest_appearances`.

Jaybone25 replied:
Hmm i'll take a look at these, this was literally just edited directly in the file on dev, I didn't generate it on stable.   So that might have already been tehre.

fenhl commented on data/Hints/scrubs.json:
```diff
@@ -32,4 +33,4 @@
     "groups": [],
     "disabled": []
-}
+}
```
The newline removed here should be added back.

Jaybone25 replied:
This is what I get for having Trent send it to me :)  I'll get that fixed

Jaybone25 replied:
I'm actually not sure what you mean here.   When compared to other presets it looks exactly the same

fenhl commented on data/presets_default.json:
```diff
@@ -417,53 +417,69 @@
+    "enhance_map_compass":                     false,
+    "mq_dungeons_random":                      false,
+    "mq_dungeons":                             0,
     "disabled_locations": [
```
For consistent style, this bracket should be indented further.

Jaybone25 replied:
Hmm, I thought it should be, but when I committed the first time, it showed that the indent was originally where i changed it back to

fenhl replied:
It was, but you changed this preset to column indentation.

Jaybone25 committed:
Various fixes to Settings/Indentation

Fix settings names to properly reflect new dev.   Fixed indentations in preset_default.json to reflect most common style.    Added linebreak back to scrubs.json

Jaybone25 commented:
All Fenhl's fixes should be addressed now

fenhl commented:
Spotted another issue: `logic_earliest_adult_trade` and `logic_latest_adult_trade` are each listed twice.

Jaybone25 committed:
Remove Duplicate Setting

Remove the biggoron earliest and latest duplicate entries in preset.

Jaybone25 commented:
Fenhl's latest fix has been addressed.

Jaybone25 committed:
Addition of Weekly Hint Distro and preset

Update to weekly preset to reflect new weekly settings.  Also addition of weekly hint distro to facilitate easier changes to weekly hints.

Jaybone25 committed:
Fixing capitalization discrepency

Fixed capitalization of "Weekly" per Fehnl's suggestion.

Jaybone25 commented:
Fixed capitalization issue

Jaybone25 commented:
When I lowercases it, it creates a syntax error.  Personally I think they should all be uppercase tbh

Sent from my iPhone

> On Jun 15, 2022, at 2:03 PM, Fenhl ***@***.***> wrote:
> 
> ﻿
> @fenhl commented on this pull request.
> 
> In data/Hints/weekly.json:
> 
> > @@ -0,0 +1,33 @@
> +{
> +    "name":                  "Weekly",
> Internal name should match the filename, i.e. lowercase w.
> 
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you authored the thread.

fenhl commented:
I'm not sure what you're referring to.
